funnel/forms/project.py

```diff
@@ -63,8 +63,14 @@ class ProjectForm(forms.Form):
         description=__(u"Eg: Explara, Instamojo"),
         validators=[forms.validators.Optional(), forms.validators.Length(max=2000)])
 
-    def validate_date_upto(self, date_upto):
-        if self.date_upto.data < self.date.data:
+    def validate_name(self, field):
+        if field.data != field.object_data:
+            # field.object_data returns the field data from the object attached to it.
+            # When a new project is being created, field.object_data is empty.
+            AvailableName()(self, field)
```

Comment:
You don't need this. AvailableName will do this for you if the form has `edit_obj`, `edit_parent` and `edit_model`.